### PR TITLE
Add some styling and new allowed HTML markup to RichTextField to support schema markup

### PR DESCRIPTION
**Pull request by Scotchester**
This is needed to experiment with some schema markup to improve search result formatting.

## Additions

- New allowed HTML elements in rich text fields: `div`, `h4`, `p`, `span`
- New allowed HTML attributes in rich text fields: `itemprop`, `itemscope`, `itemtype`

## Testing

1. Open an Ask CFPB answer for editing
1. Click the HTML editor button in its rich text field
1. Add in some of the new elements and attributes
1. Save the answer
1. Reopen the HTML editor in the RTF to see that the new stuff has not been stripped
1. Go to the answer's page and preview it
1. Inspect the answer content to see the elements are present on the rendered page

## Notes

- ~~Though I have added `div` here, for reasons I don't understand, `div` elements still get stripped~~ Explained below.

## Checklist

- [x] PR has an informative and human-readable title
- [x] Changes are limited to a single goal (no scope creep)
- [x] Code can be automatically merged (no conflicts)
- [x] Code follows the standards laid out in the [CFPB development guidelines](https://github.com/cfpb/development)
- [x] Passes all existing automated tests
- [x] Any _change_ in functionality is tested
- [x] New functions are documented (with a description, list of inputs, and expected output)
- [x] Placeholder code is flagged / future todos are captured in comments
- [x] Visually tested in supported browsers and devices (see checklist below :point_down:)
- [x] Project documentation has been updated
- [x] Reviewers requested with the [Reviewers tool](https://help.github.com/articles/requesting-a-pull-request-review/) :arrow_right:

### CODE REVIEW

File: cfgov/ask_cfpb/wagtail_hooks.py [hunk 1]
```diff
@@ -132,10 +132,16 @@ def editor_css():
 
 
 def whitelister_element_rules():
```
Comment (chosak) [inline cfgov/ask_cfpb/wagtail_hooks.py]: The placement of this hook in the `ask_cfpb` app and the title of this PR are a little misleading. These rules don't only apply to rich text fields in Ask -- they are global across the entire Wagtail site. See [the Wagtail docs](https://docs.wagtail.io/en/v1.13.4/reference/hooks.html#construct-whitelister-element-rules).

While you're modifying this, can we move this code into `v1.wagtail_hooks` to make it more obvious that it applies globally to the whole project?

Comment (Scotchester) [reply cfgov/ask_cfpb/wagtail_hooks.py]: Cool, will do. Also realizing that I need to add in a bit of Less.

**Comment by Scotchester** [comment]
> There appears to be a purposeful design decision in Wagtail rich text fields to convert <div> to <p> regardless of whitelisting rules.

Good catch. I'll remove it from the whitelist.

File: cfgov/ask_cfpb/wagtail_hooks.py (hunk 1 above)
Comment (Scotchester) [reply cfgov/ask_cfpb/wagtail_hooks.py]: Done in aa9fdf4.

File: cfgov/v1/tests/test_wagtail_hooks.py [hunk 2]
```diff
@@ -247,3 +250,100 @@ def test_tag_param_returns_resources_tagged_that_way(self):
             response.context['object_list'][1].title,
             'Test resource Banana'
         )
+
+
+class TestWhitelistOverride(TestCase):
+    # Borrowed from https://github.com/wagtail/wagtail/blob/v1.13.4/wagtail
+    # /wagtailcore/tests/test_dbwhitelister.py
+    def assertHtmlEqual(self, str1, str2):
```
Comment (chosak) [inline cfgov/v1/tests/test_wagtail_hooks.py]: Can you use the built-in Django [`assertHTMLEqual`](https://docs.djangoproject.com/en/1.11/topics/testing/tools/#django.test.SimpleTestCase.assertHTMLEqual) here? (Helpfully documented in [our docs](https://github.com/cfpb/development/blob/master/guides/unittesting-django-wagtail.md#choosing-a-testcase-base-class)!)

Comment (Scotchester) [reply cfgov/v1/tests/test_wagtail_hooks.py]: Perhaps I can!

Comment (Scotchester) [reply cfgov/v1/tests/test_wagtail_hooks.py]: Done in 4f4317f!

**Comment by virginiacc** [comment]
Thanks for tagging me, @Scotchester -- this looks good, and I'm interested to hear about the results of the experiment!

**Pull request merged**